with-go/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"context"
+	"html/template"
+	"log"
+	"net/http"
+
+	"github.com/machinebox/graphql"
+)
+
+func main() {
+	http.HandleFunc("/", getProducts)
+	http.ListenAndServe(":8080", nil)
+}
+
+func getProducts(w http.ResponseWriter, r *http.Request) {
+	client := graphql.NewClient("https://api-eu-central-1.graphcms.com/v2/ck8sn5tnf01gc01z89dbc7s0o/master")
+
+	req := graphql.NewRequest(`
+    { 
+			products {
+				slug
+				name
+			}
+		}
+	`)
+
+	ctx := context.Background()
```

Comment:
You could use r.Context here as this handles cases where the request gets aborted